lib/lsif_graph.dart

```diff
@@ -145,19 +153,30 @@ class Item extends Edge {
   Item(this.document, [this.property]) : super();
   String outV;
   List<String> inVs;
+
+  /// Where the edge is coming from - equivalent, but much easier to read than outV.
+  // TODO: Make these uniformly available.
+  // TODO: Consider pushing inV/outV up to edge, but some of them have inVs/outVs, so
+  // how do we allow for that?
+  set from(String origin) => outV = origin;
+  String get from => outV;
+
+  /// Where the edge is coming from - equivalent, but much easier to read than inVs.
+  set to(List<String> destinations) => inVs = destinations;
+  List<String> get to => inVs;
```

Comment:
#nit Might be worth calling the properties `from`/`to`, and just noting in the doc string they're equivalent to `inVs`/`outV`

Reply:
Agreed. Made an issue (#14) to standardize that and noted this as something to do there.